wordcount.c

```diff
@@ -0,0 +1,19 @@
+/* Can Parlar - Counts the words in a file */
+
+#include <stdio.h>
+
+int main() {
```

Comment:
Nice job getting it to run and work, but it didn't work quite right! It didn't work on our test file! the reason for this is you don't take all kinds of whitespace into consideration, like tabs and newlines!